cdap-ui/app/cdap/components/PluginJSONCreator/Create/index.tsx

```diff
@@ -67,6 +67,14 @@ class CreateView extends React.PureComponent<ICreateContext & WithStyles<typeof
     this.setState({ groupToInfo });
   };
 
+  public setGroupToWidgets = (groupToWidgets: any) => {
```

Comment:
Is it possible to type this? Typing this partially atleast would be helpful. Using typescript and using `any` takes away the advantage of using typescript.

Reply:
https://github.com/cdapio/cdap/pull/12251 addresses this issue by defining types more specifically! For instance, `groupToWidgets` is going to be `Map<string, List<string>>`.